taxonomy-editor-frontend/src/pages/editentry/ListEntryChildren.jsx

```diff
@@ -87,9 +88,14 @@ const ListEntryChildren = ({ url, urlPrefix, setUpdateNodeChildren }) => {
   }
   if (isPending) {
```

Comment:
@nobeeakon I am taking this PR
I can see the 2 loading spins  when we want to edit a node.
Do we want a circular progress when we are waiting for children and parents ? Or do we only need a circular progress for the whole page ?
I ask these questions to know if I delete these circular progress components in ListEntryChildren and ListEntryParents. If we want to keep them, we will display both of them or do we only want one for both lists ?

Reply:
@Piv94165 let's keep it simple ! Spinners are not the main UX blocker right now ;-)